x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/validate.ts

```diff
@@ -74,7 +75,7 @@ export const transformValidateBulkError = (
   ruleId: string,
   alert: PartialAlert,
   ruleActions?: RuleActions | null,
-  ruleStatus?: unknown
+  ruleStatus?: SavedObjectsFindResponse<IRuleStatusAttributes>
```

Comment:
This wasn't typed because some areas were passing through `ruleStatuses.saved_objects[0]` and some were passing `ruleStatuses`. Added the correct type and adjusted use cases so they all just pass in `ruleStatuses` since we were already making checks within for the existence of `.saved_objects[0]` .